TriangleApp/app/src/main/java/triangle/triangleapp/helpers/CameraHelper.java

```diff
@@ -0,0 +1,29 @@
+package triangle.triangleapp.helpers;
+
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.hardware.Camera;
+import android.util.Log;
+
+/**
+ * Created by Kevin Ly on 6/12/2017.
+ */
+
+public class CameraHelper {
+    private static final String TAG = "CameraHelper";
+
+    public static boolean checkCameraHardware(Context context) {
```

Comment:
Added comment to method: checkCameraHardware.
/**
     * Check camera exists in hardware.
     * @param context, The context of the application.
     * @return State of exists of hardware.
     */